pianoPlayer/main.js

```diff
@@ -0,0 +1,108 @@
+/* ISSUES & TODOs
+- add music!!
+- stop propagation / capture=true for when the keys are pressed (excluding the letters!)
+- mouseup event not caught when mouseout before
+- prevent key from overlapping the dark keys when it's being pressed
+- fix skeuomorphic design for upper keys
+*/
+
+// The keys and notes variables store the piano keys
+// plucks out the corresponding DOM elements and stores them as such
+const keys = ['c-key', 'd-key', 'e-key', 'f-key', 'g-key', 'a-key', 'b-key', 'high-c-key', 'c-sharp-key', 'd-sharp-key', 'f-sharp-key', 'g-sharp-key', 'a-sharp-key'];
```

Comment:
For this project, this is totally fine.

I would, however, like to call to your attention how this structure introduces a fragility in the code. Notice how _these_ strings must match _those_ id strings in the HTML markup. Whenever you add one there, you have to add one here.

_One_ of the things that frameworks like react provides, is by generating the HTML alongside the JS, a variable can more easily be defined once and then reused in 2+ places, across JS/HTML (and even CSS) to reduce the fragility of a project/feature and make it easier to tweak/reason about.